CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing Guidelines
+
+Welcome to Shipwright Build. We are excited about the prospect of you contributing to our project. Your support is more than welcome!
+
+## Getting Started
+
+We have initial documentation on how to start contributing here:
+
+- Learn how to [change shipwright and try out your changes on a local cluster](/docs/development/local_development.md)
+- Our main [documentation](/docs/)
+- Our [Code of Conduct](/code-of-conduct.md)
+
+## Creating new Issues
+
+We recommend to open an issue for the following scenarios:
+
+- Asking for help or questions. (_Use the **discussion** or **help_wanted** label_)
+- Reporting a Bug. (_Use the **bug** label_)
+- Requesting a new Feature. (_Use the **enhancement** label_)
+
+_Note_: Do not hesitate to contact us inmediately in our [Slack channel](https://kubernetes.slack.com/archives/C019ZRGUEJC).
+
+## Writing Pull Requests
+
+For making `noice` in the repository and creating new contributions, here are some recommendations:
+
+- Fork the project
+- Create a new feature branch for your contribution
+- Make your changes
+- Ensure tests are passing
+- Open a PR with a nice description and a link to the Github issue where the changes were previously discussed.
+
+## Code review process
+
+There is an integration on our Github repository that automatically do things for us. Once a PR is open the tool will assign two members of the project for the code review.
+
+The code review should cover:
+
+- Ensure all related tests(_unit, integration and e2e_) are passing.
+- Ensure the code style is compliant with the [coding conventions](https://github.com/kubernetes/community/blob/master/contributors/guide/coding-conventions.md)
+- Ensure the code is properly documented, e.g. enough comments where needed.
+- Ensure the code is adding the necessary test cases(_unit, integration or e2e_) if needed.
+
+## Community Meetings Participation
+
+We run the community meetings on Monday´s at 13:00 UTC time. For each upcoming meeting we generate a new issue where we layout the topics to discuss. See our [previous meetings](https://github.com/shipwright-io/build/issues?q=is%3Aissue+label%3Acommunity+is%3Aclosed) outcomes. For joining us, please request an invite in our Slack channel.
```

Comment:
```suggestion
We run the community meetings every Monday at 13:00 UTC time.
For each upcoming meeting we generate a new issue where we layout the topics to discuss.
See our [previous meetings](https://github.com/shipwright-io/build/issues?q=is%3Aissue+label%3Acommunity+is%3Aclosed) outcomes.
To join, please request an invite in our Slack channel.
```

Reply:
With regards to "To join, please request an invite in our Slack channel": It could be just me, but based solely on this sentence, I would not know how to request an invite to what slack channel? Maybe another link to the channel would be nice here.

Reply:
done